open_ce/build_tree.py

```diff
@@ -226,9 +226,9 @@ def __init__(self,
             validate_args.append((variant_tree, external_deps, variant_start_nodes))
 
             self._conda_env_files[variant_string] = get_conda_file_packages(variant_tree, external_deps, variant_start_nodes)
-            self._test_feedstocks[variant_string] = []
+            self._test_feedstocks[variant_string] = set()
```

Comment:
When this was a list, it would actually have multiple copies of the same feedstock and run duplicate tests.